benchmarks/launch_triton_server.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+GRPC_PORT=8001
+MODEL=$1
+
+if [ -z "$1" ]; then
+  echo "Error: Model name is missing. Please provide a value."
+  echo "The script can be run with the command \"./launch_triton_server.sh <model_name>\"."
+  exit 1
+fi
+
+set -e
+
+# Clone the Triton tutorials repository with the example vLLM deployment
+# Build the server image with the necessary dependencies
+mkdir -p triton && \
+   cd triton && \
+   rm -rf tutorials
+git clone https://github.com/triton-inference-server/tutorials.git && \
+   cd tutorials/Quick_Deploy/vLLM && \
+   docker build -t tritonserver_vllm .
+
+cd triton/tutorials/Quick_Deploy/vLLM
```

Comment:
Does this work as-is? Looks to me like:
```
&& \ cd tutorials/Quick_Deploy/vLLM  # OK
...
cd triton/tutorials/Quick_Deploy/vLLM  # maybe ERROR: Already in this directory?

Reply:
Thanks, good catch! This was a legacy of testing (was copying from one machine to another) that got missed. Removed.